src/main/java/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -1294,6 +1294,10 @@ public boolean onInteract(final Player p, MultiBlock mb, final Block b) {
 		new SoulboundItem(Categories.WEAPONS, (SlimefunItemStack) SlimefunItems.SOULBOUND_SWORD,
 		new ItemStack[] {null, SlimefunItems.ESSENCE_OF_AFTERLIFE, null, null, new ItemStack(Material.DIAMOND_SWORD), null, null, SlimefunItems.ESSENCE_OF_AFTERLIFE, null})
 		.register(true);
+		
+		new ExcludedSoulboundTool(Categories.WEAPONS, (SlimefunItemStack) SlimefunItems.SOULBOUND_TRIDENT,
```

Comment:
Why ExcludedSoulboundTool?

It's not placeable nor a tool.

Reply:
Sorry about that.